### PR TITLE
Add notes about Salt Bundle in README_TESTING

### PR DESCRIPTION
**Pull request by meaksh**
## What does this PR change?

See title

### CODE REVIEW

**Review by vandabarata — APPROVED** [review]
Thank you for this :grin:

File: README_TESTING.md [hunk 1]
```diff
@@ -102,6 +102,26 @@ You can detach from the session at anytime using the key sequence `^A d`. To re-
 ssh -t head-ctl.tf.local screen -r
 ```
 
+## Using Salt Bundle (venv-salt-minion) in "Head" and "Uyuni"
+
+Currently, our `head` and `uyuni-master` testing deployments require the Salt Bundle (venv-salt-minion package) to be installed on the each client instance and some other tunnings, before the testsuite is started. To be sure that all necessary adjustments are in place for the testsuite to run for HEAD or Uyuni, you need to set some flags on each of your instances (except for you server instance) in your TF file:
```
Comment (vandabarata) [inline README_TESTING.md:107]:
```suggestion
Currently, our `head` and `uyuni-master` testing deployments require the Salt Bundle (venv-salt-minion package) to be installed on each client instance and some other tunnings, before the testsuite is started. To be sure that all necessary adjustments are in place for the testsuite to run for HEAD or Uyuni, you need to set some flags on each of your instances (except for you server instance) in your TF file:
```

**Review by vandabarata — CHANGES_REQUESTED** [review]

File: README_TESTING.md (hunk 1 above)
Comment (vandabarata) [reply README_TESTING.md:107]: I know I initially approved but then I noticed the typo and had to comment :sweat_smile:

**Comment by nodeg** [comment]
I left some suggestions, too :wink:

File: README_TESTING.md (hunk 1 above)
Comment (meaksh) [reply README_TESTING.md:107]: Thanks! I noticed another one too sweat_smile

**Review by nodeg — APPROVED** [review]
Just some small annotations. I forgot to push the button in the review tab... Sorry.

**Pull request merged**